chain/chain/src/test_utils.rs

```diff
@@ -280,6 +280,30 @@ impl RuntimeAdapter for KeyValueRuntime {
         }
     }
 
+    fn verify_approval_signature(
```

Comment:
hm, usually in this test util KeyValueRuntime we don't verify signatures. e.g. next function just returns true

Reply:
yeah but it is required for my test which expects the signature to fail